packages/cli/tests/parsers/test_wait_create_edit_actions.py

```diff
@@ -0,0 +1,126 @@
+import json
+import pathlib
+import typing
+
+import pytest
+from click.testing import CliRunner
+
+from .data import EntityTestData, generate_entities_for_test, TRAINING, PACKAGING, DEPLOYMENT, ROUTER
+
+ENTITY_TEST_DATA: typing.Dict[str, EntityTestData] = {
+    "training": TRAINING,
+    "packaging": PACKAGING,
+    "deployments": DEPLOYMENT,
+    "routes": ROUTER,
+}
+
+#
```

Comment:
That license block seems to have broken through the edge to the main code :)

Reply:
lol, fixed